api/cds.proto

```diff
@@ -161,24 +161,16 @@ message Cluster {
   //   verification.
   UpstreamTlsContext tls_context = 11;
 
-  oneof protocol_options {
-    // [#not-implemented-hide:]
-    TcpProtocolOptions tcp_protocol_options = 12;
```

Comment:
@wora @htuch for locked protos, when we delete a field, should we do something like:
`// deleted = 12;`
or something? Just curious what the norm is here. I think something like that would help me.

Reply:
I added these comments for both 12 and 15.